### PR TITLE
Sidm 9142 Added test to add large cookie

### PR DESCRIPTION
**Pull request by arun-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/js/login_user_functional_test.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const testSuitePrefix = randomData.getRandomAlphabeticString();
 const serviceName =  randomData.getRandomServiceName(testSuitePrefix);
 const serviceClientSecret = randomData.getRandomClientSecret();
 const userPassword = randomData.getRandomUserPassword();
+const appendedStringPromise = randomData.getRandomTextFor11KB();
```
Comment (jburke-idam) [inline src/test/js/login_user_functional_test.js:18]: can this be a more descriptive name? "largeCookiePromise"? (also, why is it a promise? not sure why the get value needs to be async)

File: src/test/js/shared/random_data.js [hunk 2]
```diff
@@ -12,6 +12,15 @@ function randomAlphabeticString(length = 10) {
     return randomString
 }
 
+async function createRandomString(length) {
```
Comment (jburke-idam) [inline src/test/js/shared/random_data.js:15]: Why is this async?

File: src/test/js/login_user_functional_test.js (hunk 1 above)
Comment (arun-hmcts) [reply src/test/js/login_user_functional_test.js]: Added promise to address the protocol error on the cookie value , now after moving to text generation to glolabl declaration  i can use it without asynch

File: src/test/js/shared/random_data.js (hunk 2 above)
Comment (arun-hmcts) [reply src/test/js/shared/random_data.js]: removed

**Review by jburke-idam — APPROVED** [review]

File: src/test/js/shared/random_data.js [hunk 3]
```diff
@@ -44,6 +53,7 @@ module.exports = {
     getRandomEmailAddress: () => randomString() + "@mailtest.gov.uk",
     getRandomUserPassword: () => generatePassword(12),
     getRandomClientSecret: () => generatePassword(12),
+    getRandomTextFor11KB: () => createRandomString(1112150),
```
Comment (kremi) [inline src/test/js/shared/random_data.js:56]: I think 1112150 bytes would be ~1 MB :) 

(Also - a char is 2 bytes so we end up creating a 2MB cookie:) )

Comment (arun-hmcts) [reply src/test/js/shared/random_data.js:56]: Now its been set around 11 KB

**Pull request merged**